test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/NullRestrictedTypeOptTests.java

```diff
@@ -69,46 +82,60 @@ static public void testEAOpportunity1() throws Throwable {
 		}
 	}
 
+	@NullRestricted
+	static PrimPair evalTestEA1a_p1;
+	@NullRestricted
+	static PrimPair evalTestEA1a_p2;
+	@NullRestricted
+	static PrimPair evalTestEA1a_p3;
+	@NullRestricted
+	static PrimPair evalTestEA1a_p4;
+
 	static private void evalTestEA1a(int iter) {
 		// Test situation where EA could apply to value p1,
 		// but might have to allocate contiguously
-		PrimPair p1 = new PrimPair(1, 2);
-		PrimPair p2 = parr[0];
-
-		PrimPair p3;
-		PrimPair p4;
+		evalTestEA1a_p1 = new PrimPair(1, 2);
+		evalTestEA1a_p2 = parr[0];
 
 		if (iter % 2 == 0) {
-			p3 = p2;
-			p4 = p1;
+			evalTestEA1a_p3 = evalTestEA1a_p2;
+			evalTestEA1a_p4 = evalTestEA1a_p1;
 		} else {
-			p3 = p1;
-			p4 = p2;
+			evalTestEA1a_p3 = evalTestEA1a_p1;
+			evalTestEA1a_p4 = evalTestEA1a_p2;
 		}
-		result += p3.x + p4.y;
+		result += evalTestEA1a_p3.x + evalTestEA1a_p4.y;
 	}
 
+	@NullRestricted
+	static PrimPair evalTestEA1b_p1;
+	@NullRestricted
+	static PrimPair evalTestEA1b_p2;
+
+	@NullRestricted
+	static PrimPair evalTestEA1b_p3;
+	@NullRestricted
+	static PrimPair evalTestEA1b_p4;
+
```

Comment:
As with test `evalTestEA1a` above, making these previously local variables into static fields prevents Escape Analysis from stack allocating.  I would suggest disabling `evalTestEA1b` for now.

Reply:
As I suggested above, maybe the test can be kept active, with the static fields reverted to local variables.  There are still opportunities for Escape Analysis to perform stack allocation, despite the fact that the variables cannot be marked as null-restricted.